packages/firestore/__tests__/firestore.test.ts

```diff
@@ -679,11 +679,11 @@ describe('Firestore', function () {
       firestore1.settings({ persistence: true });
       const indexManager = firestore1.persistentCacheIndexManager();
       expect(indexManager).toBeDefined();
-      expect(indexManager.constructor.name).toEqual('FirestorePersistentCacheIndexManager');
+      expect(indexManager!.constructor.name).toEqual('FirestorePersistentCacheIndexManager');
 
-      expect(indexManager.enableIndexAutoCreation).toBeInstanceOf(Function);
-      expect(indexManager.disableIndexAutoCreation).toBeInstanceOf(Function);
-      expect(indexManager.deleteAllIndexes).toBeInstanceOf(Function);
+      expect(indexManager!.enableIndexAutoCreation).toBeInstanceOf(Function);
+      expect(indexManager!.disableIndexAutoCreation).toBeInstanceOf(Function);
+      expect(indexManager!.deleteAllIndexes).toBeInstanceOf(Function);
```

Comment:
Was it working with `?`

Reply:
It should work, but it is supposed to exist as persistence is set earlier: https://github.com/invertase/react-native-firebase/blob/fix-jest-tests/packages/firestore/__tests__/firestore.test.ts#L679

If it doesn't exist, the test should fail in any event.